From 45a5de45aa8d3a25c5a6bb3252e6c7ffc7d71897 Mon Sep 17 00:00:00 2001
From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Wed, 20 Nov 2013 13:17:21 +0100
Subject: [PATCH 1/2] Patch name to release -current version

---
 debian/control.em |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 35e0893..d41b851 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -6,7 +6,7 @@ Build-Depends: debhelper (>= 7.0.50~), @(', '.join(BuildDepends))
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
-Package: @(Package)
+Package: @(Package)-current
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 Description: @(Description)
-- 
1.7.9.5

